### ISSUE TITLE
HTML not rendered on insert

### ISSUE THREAD

**Issue by JBtje** [ratiw/vuetable-2#324]
I would like to add a tooltip to the data in the table. For this I found "vue-directive-tooltip" to be working nicely when used outside the table. However, when I send that exact string to the table as data, it does "render" the HTML, but the v-tooltip also shows up in the HTML on the page (and the tooltip is not working).
I thought that perhaps the callback could be used, but the same result so far.

    data() {
        return {
            fields: [
                ...
                {name: 'tooltip', title: this.$t('tooltip'), callback: 'test'},
    ],   }   }
    methods: {
    ...
        'test'(value) {
            return '<span v-tooltip.bottom="\'basic one\'">Tooltip</span>';
        },
_note: this.$t('tooltip') is from i18n, to translate the column title_

So basically, Vue isn't doing its magic with v-tooltip, which leaves me with the question: should this work? and if not, how can I get this to work?

**Comment by muhammadsaeedparacha** [comment]
bump. My case is I'm rendering a <router-link> and vue isn't doing its magic

**Comment by ratiw** [comment]
I don't think it's going to work. Since what returned from the callback is dynamically rendered by Vue (because of `v-html`), but it is not compiled beforehand by Vue.

**Comment by IvoPereira** [comment]
What would you recommend doing in this case @ratiw?

I was trying to implement a tooltip in a cell too.

Have you ever came to any solution @JBtje?

**Comment by JBtje** [comment]
> Have you ever came to any solution @JBtje?

I use a span with title....

**Comment by ratiw** [comment]
You should use special field instead. In v2.0, you could create a Field component to support that functionality. For a single use, Field slot (or `__slot` special field in v1.7) should be enough.

**Comment by JBtje** [comment]
For those too lazy to search: https://github.com/ratiw/vuetable-2-tutorial/wiki/lesson-11#__slotname--v120 / https://ratiw.github.io/vuetable-2/#/Special-Fields?id=-__slotltnamegt
